include/ginkgo/config.hpp.in

```diff
@@ -115,6 +115,12 @@
 // clang-format on
 
 
+/* Is LAPACK available? */
+// clang-format off
+#define GKO_HAVE_LAPACK @GINKGO_HAVE_LAPACK@
```

Comment:
```suggestion
#cmakedefine01 GINKGO_HAVE_LAPACK
```

Reply:
When I make this change, `include/ginkgo/config.hpp` seems to always define `GINKGO_HAVE_LAPACK` as 0, even when the CMake configure step reports that it is 1. What am I missing?